backend/de.metas.contracts/src/main/java/de/metas/contracts/modular/log/ModularContractLogDAO.java

```diff
@@ -0,0 +1,194 @@
+/*
+ * #%L
+ * de.metas.contracts
+ * %%
+ * Copyright (C) 2023 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.contracts.modular.log;
+
+import de.metas.bpartner.BPartnerId;
+import de.metas.calendar.standard.YearId;
+import de.metas.contracts.FlatrateTermId;
+import de.metas.contracts.model.I_ModCntr_Log;
+import de.metas.contracts.modular.settings.ModularContractTypeId;
+import de.metas.invoicecandidate.InvoiceCandidateId;
+import de.metas.lang.SOTrx;
+import de.metas.money.CurrencyId;
+import de.metas.money.Money;
+import de.metas.organization.IOrgDAO;
+import de.metas.organization.LocalDateAndOrgId;
+import de.metas.organization.OrgId;
+import de.metas.product.ProductId;
+import de.metas.quantity.Quantity;
+import de.metas.uom.IUOMDAO;
+import de.metas.util.Services;
+import lombok.NonNull;
+import org.adempiere.ad.dao.IQueryBL;
+import org.adempiere.ad.dao.IQueryBuilder;
+import org.adempiere.model.InterfaceWrapperHelper;
+import org.adempiere.util.lang.impl.TableRecordReference;
+import org.adempiere.warehouse.WarehouseId;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.Nullable;
+
+@Service
+public class ModularContractLogDAO
+{
+	private final IOrgDAO orgDAO = Services.get(IOrgDAO.class);
+	private final IUOMDAO uomDAO = Services.get(IUOMDAO.class);
+	private final IQueryBL queryBL = Services.get(IQueryBL.class);
+
+	public ModularContractLogEntry get(@NonNull final ModularContractLogEntryId id)
+	{
+		return fromDB(InterfaceWrapperHelper.load(ModularContractLogEntryId.toRepoId(id), I_ModCntr_Log.class));
+	}
+
+	public ModularContractLogEntryId create(@NonNull final LogEntryCreateRequest request)
+	{
+		final I_ModCntr_Log log = fromLogEntryCreateRequest(request);
+		InterfaceWrapperHelper.save(log);
```

Comment:
Please, use static imports for the needed methods from InterfaceWrapperHelper. For example "import static org.adempiere.model.InterfaceWrapperHelper.save;". Code is more cluttered if we use "InterfaceWrapperHelper." each time we a method from ther eis used.

Reply:
Not a fan of (ab)using static imports, unless the method names are fairly unique in the codebase or often used throughout a class. IMHO they hinder code readability outside of an IDE. But since you said please...